godtools/App/Features/Tools/SwiftUI/AllTools/ToolCategoriesView/ToolCategoryButtonView/ViewModels/AllToolsCategoryButtonViewModel.swift

```diff
@@ -9,10 +9,19 @@
 import Foundation
 
 class AllToolsCategoryButtonViewModel: BaseToolCategoryButtonViewModel {
```

Comment:
Hey @rachaelblue I would also like the All Tools category to be included in the ```GetToolCategoriesUseCase```.